modules/openapi-generator/src/main/resources/dart2/pubspec.mustache

```diff
@@ -11,8 +11,8 @@ homepage: '{{{pubHomepage}}}'
 environment:
   sdk: '>=2.0.0 <3.0.0'
```

Comment:
intl 0.17.0 minsdk is 2.12, you might want to enforce that here too

Reply:
would this generate a project that doesn't compile?

Reply:
If you use a dart < 2.12 yes, pub get will tell you that intl can be use starting 2.12

Reply:
Yes this would have to be `>=2.12.0 <3.0.0`

Reply:
The problem is, that if we update the lower bound to `>=2.12` then other dependencies will also require updates (`json_serializable`, maybe others`).